packages/neon/neon_dashboard/lib/src/blocs/dashboard.dart

```diff
@@ -109,7 +108,7 @@ class DashboardBloc extends InteractiveBloc implements DashboardBlocEvents, Dash
             Result.success(
               widgets.map(
                 (final id, final items) => MapEntry(
-                  response.body.ocs.data.values.find(id),
+                  response.body.ocs.data.values.singleWhere((final widget) => widget.id == id),
```

Comment:
Is it really important that we also verify that only a single value matches this?
To do so single has to always iterate over every element while first will stop on the first.

```suggestion
                  response.body.ocs.data.values.firstWhere((final widget) => widget.id == id),
```